README.md

```diff
@@ -93,6 +93,16 @@ db.get('foo', function (err, value) {
 })
 ```
 
+The constructor function has a `.errors` property which provides access to the different error types from [`level-errors`](https://github.com/Level/errors#api). See example below on how to use it:
+
+```js
+level('./db', { createIfMissing: false }, function (err, db) {
+  if (err instanceof level.errors.OpenError) {
+    console.log('open failed because expected db to exist')
```

Comment:
Nitpick: OpenError wraps *any* error from the underlying store, not necessarily "because expected db to exist".

Reply:
Check. I'll update the log message.